### PR TITLE
Kcw grunt/connecting views to api

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Goals
- Refactor to the Litecoin Card branch
- Show Login Capability
- Retain Registration

## Upcoming
- Forgot Password
- Virtual Card

### CODE REVIEW

**Comment by kcw-grunt** [comment]
## Note
Code Sign fails with a fat framework (combined simulator and release versions).  So for release, the PartnerAPI Framework needs to be managed just before released to the App Store.

File: loafwallet WatchKit Extension/BalanceInterfaceController.swift [hunk 1]
```diff
@@ -1,17 +1,7 @@
-//
```
[the remark this answers is not in the document]
Comment (kcw-grunt) [reply loafwallet WatchKit Extension/BalanceInterfaceController.swift:1]: Oh that's the swiftformatter.  Let me see if we can keep the headers

File: loafwallet/src/ApplicationController.swift [hunk 2]
```diff
@@ -76,318 +66,311 @@ class ApplicationController : Subscriber, Trackable {
...
                 })
-            }
+        }
 
-            group.enter()
-            LWAnalytics.logEventWithParameters(itemName:._20200111_DEDG)
-            Async.parallel(callbacks: [
-                { self.exchangeUpdater?.refresh(completion: $0) },
-                { self.feeUpdater?.refresh(completion: $0) },
-                { self.walletManager?.apiClient?.events?.sync(completion: $0) },
```
[the remark this answers is not in the document]
Comment (mosadialiou) [reply loafwallet/src/ApplicationController.swift:351]: @kcw-grunt you forgot to respond to this comment above.

[the remark this answers is not in the document]
Comment (kcw-grunt) [reply loafwallet/src/ApplicationController.swift:351]: it was just moved by the swiftformatter

[the remark this answers is not in the document]
Comment (mosadialiou) [reply loafwallet/src/ApplicationController.swift:351]: @kcw-grunt  yeah it was moved but I didn't see the call of `self.walletManager?.apiClient?.events?.sync(completion: $0) ` and `self.walletManager?.apiClient?.updateFeatureFlags(); $0()`
Thoughts?

**Pull request merged**